code/modules/vehicles/mecha/mech_fabricator.dm

```diff
@@ -142,11 +149,8 @@
 					sub_category += "Engineering"
 			else
 				sub_category += "All Cyborgs"
-		else if(ispath(built_item, /obj/item/borg_restart_board))
-			sub_category += "All Cyborgs" //Otherwise the restart board shows in the "parts" category, which seems dumb
```

Comment:
Where did this go?

Reply:
Readded it, not sure how I missed it originally